### ISSUE TITLE
Web page is not being navigated sequentially with the Tab key, Admin issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#821]
Web page is not being navigated sequentially with the Tab key, Admin issue.

**Describe the bug**
Web page is not being navigated sequentially with the Tab key, Admin issue. The sequence of focus is unordered.

**To Reproduce**

Go to Admin  (http://localhost:9992/cm/app/?view=workflow&pathType=unknown)
Use Tab key to focus on different components of the screen.
Focus is not moving in sequence on Create Site, Create Folder, Create Page and under Workflows/Roles/Users/Categories tab etc.

**Expected behavior**
Web page should be navigated sequentially with the Tab key.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
Fixed with https://github.com/percussion/percussioncms/issues/781
